addons/account/models/account_tax.py

```diff
@@ -296,6 +297,13 @@ def onchange_price_include(self):
         if self.price_include:
             self.include_base_amount = True
 
+    @api.depends('invoice_repartition_line_ids')
```

Comment:
I think you should add "amount" and "invoice_repartition_line_ids.factor" as depends